tests/models.json

```diff
@@ -67,7 +68,9 @@
             "claude-instant-1.1-100k",
             "claude-instant-1.0",
             "claude-2"
-        ]
+        ], 
+        "image":["dall-e-3",
```

Comment:
Can we remove these image models from anthropic. They are not supported

Reply:
Done